### PR TITLE
Fix charts link

### PR DESCRIPTION
**Pull request by hswolff**
Read the commits!

### CODE REVIEW

File: packages/mongo-nav/src/MongoNav.tsx [hunk 1]
```diff
@@ -220,30 +224,31 @@ function MongoNav({
...
-      activityFeed: `${hosts.cloud}/v2/${data?.currentProject?.projectId}#activity`,
-      invite: `${hosts.cloud}/v2/${data?.currentProject?.projectId}#access/add`,
+      settings: `${hosts.cloud}/v2/${currentProjectId}#settings/groupSettings`,
+      accessManager: `${hosts.cloud}/v2/${currentProjectId}#access`,
+      support: `${hosts.cloud}/v2/${currentProjectId}#info/support`,
+      integrations: `${hosts.cloud}/v2/${currentProjectId}#integrations`,
+      alerts: `${hosts.cloud}/v2/${currentProjectId}#alerts`,
+      activityFeed: `${hosts.cloud}/v2/${currentProjectId}#activity`,
+      invite: `${hosts.cloud}/v2/${currentProjectId}#access/add`,
+      charts: data?.currentProject?.chartsActivated
```
Comment (hswolff) [inline packages/mongo-nav/src/MongoNav.tsx]: This was the main addition.

**Comment by brookescarlett** [comment]
LGTM but for maintenance sake, would you mind adding a charts key to the projectNav key in the dataFixtures object?

**Comment by brookescarlett** [comment]
Two more nits:
- needs a changeset

File: packages/mongo-nav/src/MongoNav.tsx (hunk 1 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/MongoNav.tsx]: LGTM

**Comment by hswolff** [comment]
Done!

**Pull request merged**